tests/xtro-sharpie/api-annotations-dotnet/macOS-HealthKit.ignore

```diff
@@ -0,0 +1,3 @@
+# intro reports them as not present
```

Comment:
we had the same issue with iOS so we added an exception in the tests: https://github.com/xamarin/xamarin-macios/pull/18927/commits/f110664c3f06716f852363c32659ba9eb1c55d2d

should the same be done here?